apps/automation_crews/personal_assistant_crew/Readme.md

```diff
@@ -0,0 +1,37 @@
+# start Only Personal Assistant Crew
+
+## Steps to Run
+**Navigate to the Project Directory:**
+Change to the directory where the `setup.sh`, `run_crew.py`, `requirements.txt` files are located. For example:
+```shell
+cd path/to/project/directory
+```
+
+### 1. Run the Setup File
+Make the setup.sh Script Executable (if necessary):
+On Linux or macOS, you might need to make the setup.sh script executable:
+```shell
+chmod +x setup.sh
+```
+Execute the setup.sh script to set up the environment, install dependencies, login to composio and 
```

Comment:
comment not needed, its enough to say "./setup.sh" and then that file should have a comment about what it does